### ISSUE TITLE
Stream Audio Delay on RESTART

### ISSUE THREAD

**Issue by jamlensmedia** [jamlensmedia/edgeradioplayer#9]
If I open the player and play the radio station for 10 minutes or so....or any amount of time and "stop" the audio......of course the meta data keeps going.....i just stopped the audio and did not close the page.....and after 10 mins or so i clicked :"play" to hear the audio and it picked up exaclty where i had previously stopped it....10 mins ago in the middle of the news cast ......and of course the meta data was wayyy out in front.....

in short, the metadata experience vs. the audio is mismatched in many ways under many circumstances....

**Comment by jamlensmedia** [comment]
Fixed this properly, i have a quick fix available, but want to make sure name/stream is always in sync.

**Comment by jamlensmedia** [comment]
fixed in browser.  iOS may still have this issue (havent tested)
